srtcemu.cpp

```diff
@@ -3,6 +3,8 @@
  * Copyright (c) byuu
  *****/
 
+#include "snes9x.h"
+#include "srtcemu.h"
```

Comment:
Any particular reason for this? srtcemu.cpp is included in srtc.cpp and shouldn't be compiled by itself.

Reply:
That is a good catch. Nothing outside of the `macosx` directory should have been altered, and I don't recall making this change. It's possible Xcode decided to be "helpful" here. I will revert and make sure everything still compiles.

Reply:
Sure enough, that change did not need to be there. I'm removing it now.